README.md

```diff
@@ -15,6 +15,8 @@ This provider supports the following systems/architectures:
 - Linux / ARMv6
 - Windows / AMD64
 
+This provider is built on top of Terraform Protocol 6.0, which is compatible with Terraform CLI version 1.0 and later.
```

Comment:
Should we include a link to the Hashicorp Documentation?

Reply:
Added in 5c0c347532c2f7703f9b28f588d18822b6cb38a2